restApi.py

```diff
@@ -44,5 +43,20 @@ def get_watched_rooms():
    rows = c.fetchall()
    return json.dumps(results)    
 
+@app.route('/delete-watch/<int:WatchId>', methods = ['DELETE', 'OPTIONS'])
+def delete_watched_rooms(WatchId):
+   conn = sqlite3.connect("BOOKING.db")
+   conn.row_factory = sqlite3.Row
+   results = []
+   
+   c = conn.cursor()
+   try:
```

Comment:
The try is currently still passing for me even when I try to use a WatchId that does not currently exist in the database.
Looks like we will probably need to use SQL to search for the WatchId first, as you previously suggested.

I'm not sure if this is too important though, as the UI shouldn't be giving users the option to unwatch rooms that are not on their watchlist.

Reply:
I think this is satisfied in the front end